code/game/objects/buckling.dm

```diff
@@ -1,103 +1,112 @@
 /obj
-	var/can_buckle = 0
+	var/list/can_buckle = list()
```

Comment:
make this into a lazylist instead

Reply:
Lazylist doesn't seem to like typepaths, so...